standard.go

```diff
@@ -184,7 +184,7 @@ func DamageHazardsStandard(b *BoardState, settings Settings, moves []SnakeMove)
 					snake.Health = SnakeMaxHealth
 				}
 				if snakeIsOutOfHealth(snake) {
-					snake.EliminatedCause = EliminatedByOutOfHealth
+					EliminateSnake(snake, EliminatedByOutOfHealth, "", b.Turn+1)
```

Comment:
I apply `b.Turn+1` because `b` should be the previous board state. If we eliminate a snake at this iteration, the turn that the snake is eliminated on should be the previous state's turn incremented by 1. I think... right?